src/main/kotlin/app/extractors/CSSExtractor.kt

```diff
@@ -0,0 +1,44 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Liubov Yaronskaya (lyaronskaya@sourcerer.io)
+
+package app.extractors
+
+import app.model.CommitStats
+import app.model.DiffFile
+
+class CssExtractor : ExtractorInterface {
+    companion object {
+        val LANGUAGE_NAME = "css"
+        val FILE_EXTS = listOf("css", "scss", "less")
+    }
+
+    override fun extract(files: List<DiffFile>): List<CommitStats> {
+        files.map { file -> file.language = LANGUAGE_NAME }
+        val lessNumLinesAdded = files.filter { it.extension == "less" }
```

Comment:
Would be nice to fold this code into a function, like:
(lessNumAdded, lessNumDeleted) = getNumsWithExtensions(LESS_EXT)